awx/api/templates/api/job_template_launch.md

```diff
@@ -26,19 +26,15 @@ The response will include the following fields:
   job_template (array, read-only)
 * `survey_enabled`: Flag indicating whether the job_template has an enabled
   survey (boolean, read-only)
-* `credential_needed_to_start`: Flag indicating the presence of a credential
```

Comment:
This is still returned in the API response, but is always `false`, because now credentials are never required to launch a JT.